samples/apps/bookinfo/rules/mixer-rule-ratings-ratelimit.yaml

```diff
@@ -1,8 +1,37 @@
-rules:
-- aspects:
-  - kind: quotas
-    params:
-      quotas:
-      - descriptorName: RequestCount
-        maxAmount: 1
-        expiration: 1s
+apiVersion: "config.istio.io/v1alpha2"
+kind: memquota
+metadata:
+  name: handler
+  namespace: istio-config-default
+spec:
+  quotas:
+  - name: requestcount.quota.istio-config-default
+    max_amount: 5000
+    valid_duration: 1s
+    overrides:
+    - dimensions:
+        destination: ratings
```

Comment:
Might be worth adding a comment what this means

Reply:
done